2011/12/22 03:51:55 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[HY000] [2002] Can't connect to local MySQL server through socket '/Applications/XAMPP/xamppfiles/var/mysql/mysql.sock' (2)' in /Applications/XAMPP/xamppfiles/htdocs/pipeworks-code/yii-1.1.8.r3324/framework/db/CDbConnection.php:364
Stack trace:
#0 /Applications/XAMPP/xamppfiles/htdocs/pipeworks-code/yii-1.1.8.r3324/framework/db/CDbConnection.php(313): CDbConnection->open()
#1 /Applications/XAMPP/xamppfiles/htdocs/pipeworks-code/yii-1.1.8.r3324/framework/db/CDbConnection.php(291): CDbConnection->setActive(true)
#2 /Applications/XAMPP/xamppfiles/htdocs/pipeworks-code/yii-1.1.8.r3324/framework/base/CModule.php(372): CDbConnection->init()
#3 /Applications/XAMPP/xamppfiles/htdocs/pipeworks-code/yii-1.1.8.r3324/framework/base/CModule.php(86): CModule->getComponent('db')
#4 /Applications/XAMPP/xamppfiles/htdocs/pipeworks-code/yii-1.1.8.r3324/framework/gii/generators/model/ModelCode.php(54): CModule->__get('db')
#5 /Applications/XAMPP/xamppfiles/htdocs/pipeworks-code/yii-1.1.8.r3324/framework/web/CFormModel.php(40): ModelCode->init()
#6 /Applications/XAMPP/xamppfiles/htdocs/pipeworks-code/yii-1.1.8.r3324/framework/gii/CCodeGenerator.php(149): CFormModel->__construct()
#7 /Applications/XAMPP/xamppfiles/htdocs/pipeworks-code/yii-1.1.8.r3324/framework/gii/CCodeGenerator.php(61): CCodeGenerator->prepare()
#8 /Applications/XAMPP/xamppfiles/htdocs/pipeworks-code/yii-1.1.8.r3324/framework/web/actions/CInlineAction.php(50): CCodeGenerator->actionIndex()
#9 /Applications/XAMPP/xamppfiles/htdocs/pipeworks-code/yii-1.1.8.r3324/framework/web/CController.php(300): CInlineAction->runWithParams(Array)
#10 /Applications/XAMPP/xamppfiles/htdocs/pipeworks-code/yii-1.1.8.r3324/framework/web/CController.php(278): CController->runAction(Object(CInlineAction))
#11 /Applications/XAMPP/xamppfiles/htdocs/pipeworks-code/yii-1.1.8.r3324/framework/web/CController.php(257): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /Applications/XAMPP/xamppfiles/htdocs/pipeworks-code/yii-1.1.8.r3324/framework/web/CWebApplication.php(277): CController->run('')
#13 /Applications/XAMPP/xamppfiles/htdocs/pipeworks-code/yii-1.1.8.r3324/framework/web/CWebApplication.php(136): CWebApplication->runController('gii/model')
#14 /Applications/XAMPP/xamppfiles/htdocs/pipeworks-code/yii-1.1.8.r3324/framework/base/CApplication.php(158): CWebApplication->processRequest()
#15 /Applications/XAMPP/xamppfiles/htdocs/pipeworks-code/tweets/index.php(13): CApplication->run()
#16 {main} REQUEST_URI=/pipeworks-code/tweets/index.php?r=gii/model
2011/12/22 03:52:32 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[HY000] [2005] Unknown MySQL server host 'http://localhost' (1)' in /Applications/XAMPP/xamppfiles/htdocs/pipeworks-code/yii-1.1.8.r3324/framework/db/CDbConnection.php:364
Stack trace:
#0 /Applications/XAMPP/xamppfiles/htdocs/pipeworks-code/yii-1.1.8.r3324/framework/db/CDbConnection.php(313): CDbConnection->open()
#1 /Applications/XAMPP/xamppfiles/htdocs/pipeworks-code/yii-1.1.8.r3324/framework/db/CDbConnection.php(291): CDbConnection->setActive(true)
#2 /Applications/XAMPP/xamppfiles/htdocs/pipeworks-code/yii-1.1.8.r3324/framework/base/CModule.php(372): CDbConnection->init()
#3 /Applications/XAMPP/xamppfiles/htdocs/pipeworks-code/yii-1.1.8.r3324/framework/base/CModule.php(86): CModule->getComponent('db')
#4 /Applications/XAMPP/xamppfiles/htdocs/pipeworks-code/yii-1.1.8.r3324/framework/gii/generators/model/ModelCode.php(54): CModule->__get('db')
#5 /Applications/XAMPP/xamppfiles/htdocs/pipeworks-code/yii-1.1.8.r3324/framework/web/CFormModel.php(40): ModelCode->init()
#6 /Applications/XAMPP/xamppfiles/htdocs/pipeworks-code/yii-1.1.8.r3324/framework/gii/CCodeGenerator.php(149): CFormModel->__construct()
#7 /Applications/XAMPP/xamppfiles/htdocs/pipeworks-code/yii-1.1.8.r3324/framework/gii/CCodeGenerator.php(61): CCodeGenerator->prepare()
#8 /Applications/XAMPP/xamppfiles/htdocs/pipeworks-code/yii-1.1.8.r3324/framework/web/actions/CInlineAction.php(50): CCodeGenerator->actionIndex()
#9 /Applications/XAMPP/xamppfiles/htdocs/pipeworks-code/yii-1.1.8.r3324/framework/web/CController.php(300): CInlineAction->runWithParams(Array)
#10 /Applications/XAMPP/xamppfiles/htdocs/pipeworks-code/yii-1.1.8.r3324/framework/web/CController.php(278): CController->runAction(Object(CInlineAction))
#11 /Applications/XAMPP/xamppfiles/htdocs/pipeworks-code/yii-1.1.8.r3324/framework/web/CController.php(257): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /Applications/XAMPP/xamppfiles/htdocs/pipeworks-code/yii-1.1.8.r3324/framework/web/CWebApplication.php(277): CController->run('')
#13 /Applications/XAMPP/xamppfiles/htdocs/pipeworks-code/yii-1.1.8.r3324/framework/web/CWebApplication.php(136): CWebApplication->runController('gii/model')
#14 /Applications/XAMPP/xamppfiles/htdocs/pipeworks-code/yii-1.1.8.r3324/framework/base/CApplication.php(158): CWebApplication->processRequest()
#15 /Applications/XAMPP/xamppfiles/htdocs/pipeworks-code/tweets/index.php(13): CApplication->run()
#16 {main} REQUEST_URI=/pipeworks-code/tweets/index.php?r=gii/model
2011/12/22 04:15:35 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "list".' in /Applications/XAMPP/xamppfiles/htdocs/pipeworks-code/yii-1.1.8.r3324/framework/web/CController.php:477
Stack trace:
#0 /Applications/XAMPP/xamppfiles/htdocs/pipeworks-code/yii-1.1.8.r3324/framework/web/CController.php(262): CController->missingAction('list')
#1 /Applications/XAMPP/xamppfiles/htdocs/pipeworks-code/yii-1.1.8.r3324/framework/web/CWebApplication.php(277): CController->run('list')
#2 /Applications/XAMPP/xamppfiles/htdocs/pipeworks-code/yii-1.1.8.r3324/framework/web/CWebApplication.php(136): CWebApplication->runController('tweet/list')
#3 /Applications/XAMPP/xamppfiles/htdocs/pipeworks-code/yii-1.1.8.r3324/framework/base/CApplication.php(158): CWebApplication->processRequest()
#4 /Applications/XAMPP/xamppfiles/htdocs/pipeworks-code/tweets/index.php(13): CApplication->run()
#5 {main} REQUEST_URI=/pipeworks-code/tweets/index.php?r=tweet/list
